### ISSUE TITLE
Reproducing the results

### ISSUE THREAD

**Issue by chandlerbing65nm** [FishMaster93/U-FFIA#4]
Hi again. I'm trying to reproduce the results in Table 1 from the [paper](https://arxiv.org/pdf/2309.05058.pdf
). What [configs](https://github.com/FishMaster93/U-FFIA/tree/main/config/audio) in the yml files should I use to reproduce the `Pre-MobileNetV2`, `Pre-CNN10`, and `Pre-CNN6` results from Table 1.

**Comment by FishMaster93** [comment]
you can use the config/audio/exp1.yaml reproduce the  Pre-MobileNetV2, Pre-CNN10, and Pre-CNN6 results from Table 1.

**Comment by chandlerbing65nm** [comment]
@FishMaster93 I'm a bit confused by the configs and code structure. For table 1 result of `Pre-CNN10` and `Pre-MobileNetV2`, what should I set as:

```python
Model: ?
Model_pretrained: ?
```

and in the pretrained files, what files should I [choose](https://drive.google.com/drive/folders/1fh-Lo3S7-aTgfPni5-IeG5_-P7MBKBfL) for preloading the weights? I've tried mutiple files here conrresponding the the model and pre-model, but it doesn't work.

**Comment by FishMaster93** [comment]
I have changed the code and given you an example of how to use the pre-trained model Cnn10, the config/audio/pre_exp.yam is the pre-trained Cnn10, and in the main.py look at the "model = AudioModel_pre_Cnn10(frontend=frontend, backbone=backbone)" this is a pre-trained function, you can find it in the models/Audio_model. now you can run it, but I cannot change all code for you, you may read those code and try to understand it.

**Comment by chandlerbing65nm** [comment]
@FishMaster93 Thanks for the help. I have run the training for the config you gave but the result is far from the published result in Table 1 for 'Pre-CNN10'. 

I've got this test result after training: '2024-07-20 13:09:52,597 - INFO -  test_dataset mAP: 0.7015687491004299, accuracy: 0.7271428571428571'

I've also attached the log file for the training for full reference: [Cnn10-Pretrained-1721464765.log](https://github.com/user-attachments/files/16321779/Cnn10-Pretrained-1721464765.log)

**Comment by FishMaster93** [comment]
It is due to your dataset not being enough, the training dataset should be 213000, not 3200. I don't know what happened. I have updated the code. This is the result of my new training of 80 epochs and accuracy is 0.83. It is not fully trained yet, and it can be improved 
[Pretrained-Cnn10-1721506338.log](https://github.com/user-attachments/files/16322403/Pretrained-Cnn10-1721506338.log)
. You should check whether the dataset you downloaded is enough for 27067 clips.
